core/consensus/sumeragi.cpp

```diff
@@ -267,11 +261,11 @@ namespace sumeragi {
 
         logger::info("sumeragi")    <<  "hash:" <<  event.transaction().hash();
         logger::info("sumeragi")    <<  "pub:"  <<  config::PeerServiceConfig::getInstance().getMyPublicKey();
-        logger::info("sumeragi")    <<  "pro:"  <<  config::PeerServiceConfig::getInstance().getPrivateKey();
+        logger::info("sumeragi")    <<  "pro:"  <<  config::PeerServiceConfig::getInstance().getMyPrivateKey();
```

Comment:
Sorry, This is my mistake, 
Could you modify it "pro:" -> "pri" or "private key:"  🙇